homeassistant/components/enocean/sensor.py

```diff
@@ -25,34 +27,44 @@
 
 DEFAULT_NAME = "EnOcean sensor"
 
-DEVICE_CLASS_POWER = "powersensor"
+SENSOR_TYPE_HUMIDITY = "humidity"
+SENSOR_TYPE_POWER = "powersensor"
+SENSOR_TYPE_TEMPERATURE = "temperature"
+SENSOR_TYPE_WINDOWHANDLE = "windowhandle"
 
 SENSOR_TYPES = {
-    DEVICE_CLASS_HUMIDITY: {
+    SENSOR_TYPE_HUMIDITY: {
         "name": "Humidity",
         "unit": "%",
         "icon": "mdi:water-percent",
         "class": DEVICE_CLASS_HUMIDITY,
     },
-    DEVICE_CLASS_POWER: {
+    SENSOR_TYPE_POWER: {
         "name": "Power",
         "unit": POWER_WATT,
         "icon": "mdi:power-plug",
-        "class": DEVICE_CLASS_POWER,
```

Comment:
We should be able to use `DEVICE_CLASS_POWER` imported from the sensor integration.

https://developers.home-assistant.io/docs/en/entity_sensor.html#available-device-classes

https://github.com/home-assistant/home-assistant/blob/36f7096f0959b374b1c646493f22eb7aa9d0a771/homeassistant/components/sensor/__init__.py#L41

Reply:
@MartinHjelmare, I guess it is ok, if I import DEVICE_CLASS_POWER from const.py like DEVICE_CLASS_TEMPERATURE and DEVICE_CLASS_HUMIDITY. Correct?

Reply:
Yes, that's ok.